test/Common/LogEventTraceListener.cs

```diff
@@ -129,7 +129,7 @@ private static bool ShouldExcludeEvent(TraceEvent traceEvent, IDictionary<string
         public void Stop()
         {
             this.currentSession?.Source.StopProcessing();
-            this.backgroundTraceThread.Join(TimeSpan.FromSeconds(10));
+            this.backgroundTraceThread?.Join(TimeSpan.FromSeconds(10));
```

Comment:
Was this causing some exceptions in some cases?

Reply:
The ETW tracing was breaking for me so I had to disable it by commenting out some code. Commenting out the code caused this null-ref. I felt I should leave this null-check in, however, because in theory the `Stop()` method could get called (e.g. via `Dispose()`) before this variable is initialized, so it's actually a correct thing to check.